src/core/pyproxy.js

```diff
@@ -17,7 +17,7 @@
  * See Makefile recipe for src/js/pyproxy.js
  */
 
-import { Module } from "../js/module.js";
+import { Module } from "./module.js";
```

Comment:
This will mess up the PyProxy API docs.

Reply:
I guess we can make a symlink `pyproxy.js` ==> `../core/pyproxy.gen.js`.

Reply:
Actually the output it doesn't seem to be messed up so I guess this change is fine by itself.